docs/internals/frontends.rst

```diff
@@ -152,8 +245,222 @@ username
 comment
     Archive comment, if any
 
+Example of a simple archive listing (``borg list --last 1 --json``)::
+
+    {
+        "archives": [
+            {
+                "archive": "2017-02-27T21:21:51",
+                "barchive": "2017-02-27T21:21:51",
+                "id": "80cd07219ad725b3c5f665c1dcf119435c4dee1647a560ecac30f8d40221a46a",
+                "name": "2017-02-27T21:21:51",
```

Comment:
^- these three:

archive/barchive is the usual deal we have with POSIX (like path/bpath), just pointless for JSON since we don't get non-text anyway. We could do something more invasive here, like base64, but I don't see much point for now. Note that archive/barchive/time are not documented, only start is. The former three come from ArchiveFormatter which is used for the regular listings, but we use start here, since that makes the formats supersets of each other: list-archive-format < create-archive-format < info-archive-format

The latter two could be made the same with some more fiddling; the difference only lies in what the caller knows anyway though.